.github/workflows/python-checks.yml

```diff
@@ -62,65 +62,6 @@ jobs:
       - name: Dataset unit tests
```

Comment:
do we still need this file?

Reply:
Yes. It still contains our unit tests for Python, and our Python linting